internal/cmd/run/xcuitest.go

```diff
@@ -102,6 +102,9 @@ func runXcuitest(cmd *cobra.Command, xcuiFlags xcuitestFlags, isCLIDriven bool)
 	if err := xcuitest.Validate(p); err != nil {
 		return 1, err
 	}
+	if err := xcuitest.ShardSuites(&p); err != nil {
```

Comment:
We'll also need this as a flag for the CLI driven commands.

Reply:
Will add.